scripts/jenkins/ardana/gerrit/gerrit_merge.py

```diff
@@ -24,6 +24,43 @@ def check_all_dependencies_satisfied(change):
 
 
 def gerrit_merge(change):
+    """
+    Attempt to merge a Gerrit change.
+
+    The following criteria must be met first for a Gerrit change to
+    be merged:
+
+    1. the 'status' value must be NEW (not MERGED and ABANDONED, obviously)
+    2. the 'mergeable' flag must be set, meaning there are no merge conflicts
+    between the Gerrit change and the target branch.
+    3. the 'submittable' flag must be set, meaning that the change has been
+    approved by the project submit rules when:
+
+      - at least one Code-Review+2 label value
+      - no Code-Review-2 label values
+      - at least one Verified+2 label value
+      - no Verified-2 label value
+      - at least one Workflow+1 label value
+      - no Workflow-1 label value
+
+    NOTE: if this change is submitted together with other changes, the
```

Comment:
(nit) I think you are mixing terminologies of submitted here. Gerrit uses the term "submit" to mean "submit to the repo" or "Merge this now". However, I think you've also used it in meaning "uploaded to gerrit" throughout this document which may cause some confusion.

Reply:
done